lib/opencensus/trace/formatters/trace_context.rb

```diff
@@ -41,15 +41,15 @@ class TraceContext
         #
         # @private
         #
-        HEADER_NAME = "Trace-Context".freeze
+        HEADER_NAME = "Trace-Parent".freeze
```

Comment:
traceparent is one word, without dash

Reply:
I checked that using `Trace-Parent` in HTTP request header key has deviated from the specification.
I apologize to you for my misunderstanding.

I would like to confirm just in case.
If propagate distributed tracing context with HTTP then we have to use `traceparent` in header key in according with [this specification](https://github.com/w3c/distributed-tracing/blob/7e7e7dfa8f621c86619966be497662f4a76a3598/trace_context/HTTP_HEADER_FORMAT.md), right?